other/ingress_host_match_tls/ingress-host-match-tls.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     policies.kyverno.io/severity: medium
     kyverno.io/kyverno-version: 1.6.0
     policies.kyverno.io/minversion: 1.6.0
-    kyverno.io/kubernetes-version: "1.23"
+    kyverno.io/kubernetes-version: "1.20, 1.21, 1.23"
```

Comment:
So you did in fact test this modified policy across all three of these versions? In the webhook? And it worked?

Reply:
I tested this modified policy on 1.20 and 1.21 only.

Reply:
Should I remove the mention for 1.23?

Reply:
I just removed the mention.